spec/integration/user_can_place_first_bid_on_an_item_spec.rb

```diff
@@ -16,4 +16,21 @@
     expect(page).to have_content("Highest Bidder: #{user.name}")
     expect(page).to have_content("Highest Bid: #{user.bids.last.price}")
   end
+
+  scenario "user cannot place the same bid twice" do
+    user = create(:user)
+    item = create(:item)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+    visit item_path(item)
+
+    expect(page).to have_content("No current bids")
+    fill_in "bid[price]", with: "10.99"
+    click_button("Place Bid")
+
```

Comment:
The view blocks the entry of a duplicate bid. Since this is going through the view, it isn't checking the model validations. I think we need to change this